frontend/src/static/js/main.js

```diff
@@ -115,6 +116,16 @@ window.app = new window.Vue({
 
     errorMessages: {},
   },
+  watch: {
+    '$store.state.zoomTabInfo': function () {
```

Comment:
hmm, with these two additions, is the `updateTabZoom` and `updateTabAuthorship` methods still necessary?

Reply:
it was removed, accidentally added back after merging latest master